.github/workflows/cleanup-staging.yml

```
@@ -0,0 +1,43 @@
+name: Cleanup staging URLs
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@0.9.0
+        id: comment
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const isValidPreview = (
+              context.payload.issue.pull_request &&
+              context.payload.comment.body === '/preview'
+            );
+            return isValidPreview;
+
+      - name: Setup GCloud
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_email: ${{ secrets.GCP_SA_EMAIL }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+
+      - name: Delete preview on GCS
+        run: gsutil rm gs://staging.nodejs.dev/${{ github.event.issue.number }}/
```

```suggestion
        run: gsutil -m rm -r gs://staging.nodejs.dev/${{ github.event.issue.number }}/
```

`-m` will let it run in parallel `-r` is necessary to recursively delete a directory